backend/src/api/matches/matches.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { MatchesController } from './matches.controller';
+
+describe('MatchesController', () => {
+  let controller: MatchesController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
```

Comment:
Is it for Unit Testing ? Interesting :bulb:

Reply:
Yes I think so. It create the spec file automaticly when you create a controller a service file with the `nest generate` command. Haven't done any research on it yet